### PR TITLE
Add support matrix

### PR DESCRIPTION
**Pull request by dmpop**
### PR creator: Description

Add container support matrix. Rename _odg_masters_ to _originals_

### PR creator: Are there any relevant issues/feature requests?

* SLE-17931

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(if applicable to 15 SP4 _only_ -- do not merge yet!)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3 *(current `main`, no backport necessary)*
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/containers-support.xml [hunk 1]
```diff
@@ -9,44 +9,68 @@
...
+    <imageobject role="fo">
+     <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
+    </imageobject>
+   </mediaobject>
+  </informalfigure>
+  <para>
+  In most scenarios, all SLE containers are expected to be interoperable if the application or its dependencies do not interact directly with kernel version-specific data structures (<literal>ioctl</literal>, <literal>/proc</literal>, <literal>/sys</literal>, <literal>routing</literal>, <literal>iptables</literal>, <literal>nftables</literal>, <literal>eBPF</literal>, etc.) or modules (KVM, OVS, SystemTap, etc.). Support for <literal>ioctl</literal> and access to <literal>/proc</literal> is limited to the most common scenarios needed by unprivileged uses.
```
Comment (lproven) [inline xml/containers-support.xml:33]:
```suggestion
  In most scenarios, all SLE containers are expected to be interoperable if the
  application or its dependencies do not interact directly with kernel version-specific
  data structures (<literal>ioctl</literal>, <literal>/proc</literal>, <literal>/sys</literal>,
  <literal>routing</literal>, <literal>iptables</literal>, <literal>nftables</literal>,
  <literal>eBPF</literal>, etc.) or modules (KVM, OVS, SystemTap, etc.).
  Support for <literal>ioctl</literal> and access to <literal>/proc</literal> is limited
  to the most common scenarios needed by unprivileged uses.
```

Comment (lproven) [reply xml/containers-support.xml:33]: I line-wrapped this so I could read it all.

Unprivileged _uses_ or _users_?

File: xml/containers-support.xml [hunk 2]
```diff
@@ -179,13 +203,14 @@
...
-       supports components that come from the &suse; base containers. Packages
-       from &suse; repositories are also supported. Additional components and
-       application in the containers are not covered by &suse; support. A
-       &slea; subscription is required for building a derived container.
+       This tier targets environments delivered by selected third-party
+       vendors. While &suse;-based containers are fully supported, issues in
+       the host environment must be handled by the host environment vendor.
+       &suse; supports components that come from the &suse; base containers.
+       Packages from &suse; repositories are also supported. Additional
+       components and application in the containers are not covered by &suse;
```
Comment (lproven) [inline xml/containers-support.xml:211]:
```suggestion
       components and applications in the containers are not covered by &suse;
```

**Review by lproven — CHANGES_REQUESTED** [review]

**Comment by dmpop** [comment]
I've just realized that conflating two different things (new dir and new contents) into one PR wasn't a good idea. I'm going to close this PR and create two separate ones.

**Pull request closed without merging**